slides/src/content/routing.md

```diff
@@ -189,109 +217,247 @@ export class ProductDetails {
 
 Notes:
 
-The reason that the `params` property on `ActivatedRoute` is an Observable is that the router may not recreate the component when navigating to the same component. In this case the parameter may change without the component being recreated.
+The reason that the `params` property on `ActivatedRoute` is an Observable is that the router may not recreate the component when navigating to that same component. In this case the parameters may change without the component being recreated.
 
 ---
 
-## Child Routes (1/3)
+## Passing Optional Parameters (1/2)
+
+Use the `[queryParams]` directive along with `[routerLink]` to pass query parameters. For example:
 
-Specify child routes by using the `children` route property.
+```html
+<a [routerLink]="['/product-list']" [queryParams]="{ page: 99 }">Go to Page 99</a>
+```
```

Comment:
Wouldn't hurt to show the resulting URL here. e.g. localhost:3000/product-list?page=99